screens/signIn/index.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import { Text, View } from 'react-native';
+
+import { connect } from 'react-redux';
+import { login } from '../../actions/session';
+import styles from './styles';
+import Background from '../common/background';
+
+import Input from '../../components/common/input';
+import Button from '../../components/common/button';
+
+
+class SignIn extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      username: "",
+      password: "",
+    }
+  }
+  handleInput = (inputName, text) => this.setState({ [inputName]: text });
+  handleSubmit = () => {
+    const { username, password } = this.state;
+    const { signIn } = this.props;
+    console.log('PRELOGIN');
+    const user = { username: username, password: password };
```

Comment:
You can pass thought this json in this way.
```suggestion
    const user = { username , password };
```